ReactAndroid/src/main/java/com/facebook/react/uimanager/ViewManagerPropertyUpdater.java

```diff
@@ -13,7 +13,7 @@
 
 public class ViewManagerPropertyUpdater {
   public interface Settable {
-    Map<String, String> getProperties();
+    Map<String, String> getProperties(Map<String, String> map);
```

Comment:
This is confusing, it's adding to the map, not getting the properties. Make this code clearer.